pkg/scheduler/cache/util.go

```diff
@@ -76,3 +79,114 @@ func createImageExistenceMap(nodes []*v1.Node) map[string]sets.String {
 	}
 	return imageExistenceMap
 }
+
+// ProtocolPort represents a protocol port pair, e.g. tcp:80.
+type ProtocolPort struct {
```

Comment:
rather than adding this to utils.go, what do you think about creating a file like ports.go (and ports_test.go)?

Reply:
DONE.